src/transformState.js

```diff
@@ -5,7 +5,31 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const ADD_PROPERTIES = 'addProperties';
+  const REMOVE_PROPERTIES = 'removeProperties';
+  const CLEAR_PROPERTIES = 'clear';
```

Comment:
why you hard coded all this type's instead of using switch case?

Reply:
[checklist](https://github.com/mate-academy/js_stateful-object/blob/master/checklist.md) rule 2